crates/sui-indexer-alt-schema/migrations/2025-05-19-231614_consistent-pipelines-deletion-reference-table/up.sql

```diff
@@ -0,0 +1,28 @@
+CREATE TABLE IF NOT EXISTS obj_info_deletion_reference
+(
+    object_id                   BYTEA         NOT NULL,
+    cp_sequence_number          BIGINT        NOT NULL,
```

Comment:
I keep forgetting that because we only store at checkpoint granularity, and we only store ownership changes, we can't just track the immediate predecessor :(